### ISSUE TITLE
OCI runtime create failed

### ISSUE THREAD

**Issue by dimidd** [eliorc/denzel#1]
After installing denzel in a virtualenv (on amazon linux DL image) and running
` denzel startproject --gpu test_denzel` I get the following error:

```
ERROR: for api  Cannot start service api: OCI runtime create failed: container_linux.go:348: starting container process caused "process_linux.go:402: container init caused \"rootfs_linux.go:58: mounting \\\"/etc/timezone\\\" to rootfs \\\"/var/lib/docker/overlay2/57eb4b66dd5a34ff86d7d9ce55b3f9d1f2fb37e236fd3ef3350970520e885402/merged\\\" at \\\"/var/lib/docker/overlay2/57eb4b66dd5a34ff86d7d9ce55b3f9d1f2fb37e236fd3ef3350970520e885402/merged/etc/timezone\\\" caused \\\"not a directory\\\"\"": unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
ERROR: Encountered errors while bringing up the project.
```

My `/etc/timezone` is a dir.
Might be related to https://github.com/eliorc/denzel/blob/master/src/denzel/docker-compose.yml#L14

**Comment by eliorc** [comment]
Could you supply steps to replicate? `startproject` does not actually run anything, just creates the skeleton.

What were the steps you made that lead to this error ?

**Comment by dimidd** [comment]
Sorry, I forgot to mention the last step: `denzel launch`, which triggered the error.
To reproduce:
1. spin an instance of AWS deep learning AMI on a GPU sever with Amazon Linux
2. `activate` the tensorflow36 virtualenv
3. `pip install denzel`
4. `denzel startproject --gpu test_denzel`
5.  `cd test_denzel`
6.  `denzel launch`

**Comment by eliorc** [comment]
Could you please go on the shell of your AWS instance, and show me the output of `ls -l /etc`?

I want to verify that AWS Linux (which I know is custom made) indeed has these two directories - because I don't seem to get this error replicating these steps on my local machine

**Comment by dimidd** [comment]
Sure
```
$ ls -la /etc/{timezone,localtime}
-rw-r--r-- 1 root root  118 Aug 11 01:27 /etc/localtime

/etc/timezone:
total 8
drwxr-xr-x  2 root root 4096 Oct 22 09:53 .
drwxr-xr-x 93 root root 4096 Oct 22 09:53 ..
```

**Comment by eliorc** [comment]
Yeah, I guess that is the issue, as you said your `/etc/timezone` is a directory, which is supposed to be a file AFAIK.

I believe it might be redundant to have them both connected to the system (`timezone` and `localtime`) and I will verify that it works without `timezone` - if so, I will remove that from the mounted volumes.

For the meantime you can just remove the line that contains the timezone mounting from the `docker-compose.yml`, exactly where you linked to in the original post.

**Comment by dimidd** [comment]
Thank you! I've opened a PR to trigger the CI tests.

**Comment by eliorc** [comment]
Solved at v1.0.5
